specification/marketplacecatalog/resource-manager/Solutions/preview/2023-03-01-preview/examples/GetUserSolution.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "solutionId": "7da626db-4731-4565-bda7-718a052f2219",
+    "api-version": "2023-03-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "7da626db-4731-4565-bda7-718a052f2219",
+        "type": "Microsoft.Marketplace/solutions",
```

Comment:
>solutions [](http://example.com/codeflow?start=39&length=9)

the resource type in the spec is "Microsoft.Marketplace/mySolutions"

Reply:
done